spec/controllers/v1/sessions_controller_spec.rb

```diff
@@ -1099,21 +1174,24 @@ def expect_logger_msg(level, msg)
           end
 
           it 'creates an Account record for the user' do
-            post :saml_callback
+            call_endpoint
 
             expect(Account.first.idme_uuid).to eq uuid
           end
         end
 
         context 'and the current user already has an Account record' do
           it 'does not create a new Account record for the user', :aggregate_failures do
-            post :saml_callback
+            call_endpoint
 
             expect(Account.count).to eq 1
-            expect(Account.first.idme_uuid).to eq loa3_user.idme_uuid
+            expect(Account.first.idme_uuid).to eq user.idme_uuid
           end
         end
       end
     end
   end
+
+  describe 'GET #metadata' do
```

Comment:
No Specs for metadata?

Reply:
Nope, that was one of the benefits of this refactor; it became super clear where the gaps in coverage were. I held off on adding coverage for the metadata for now, but I am considering doing it in a fast follow. I left it as an empty block for now to call out the fact there is no coverage for it.